### PR TITLE
CEDS-2295 Insert 3rd bullet point about declarations clearing

### PR DESCRIPTION
**Pull request by PauloPortugal**
Add statement:
"Declarations received after 1500 hrs will be cleared by 0800 hrs the following day"

### CODE REVIEW

**Comment by PauloPortugal** [comment]
please test again

**Comment by PauloPortugal** [comment]
test again please

**Comment by PauloPortugal** [comment]
test again

**Comment by PauloPortugal** [comment]
test this please

File: conf/messages [hunk 1]
```diff
@@ -98,10 +98,11 @@ fileUploadReceiptPage.whatHappensNext=What happens next?
 fileUploadReceiptPage.paragraph1=As long as you have provided the documents that HMRC need to clear your goods, they should be cleared within these time limits:
 fileUploadReceiptPage.listitem1=2 hours for air freight imports, road freight imports, all exports
 fileUploadReceiptPage.listitem2=3 hours (between 0800 and 1500) for maritime freight imports
+fileUploadReceiptPage.listitem3=Declarations received after 1500 hrs will be cleared by 0800 hrs the following day
```
Comment (PauloPortugal) [inline conf/messages:101]: This was the bullet that we need to re-insert.

**Review by bethany-kish — APPROVED** [review]

**Pull request merged**